libs/ui/src/lib/tooltip/tooltip.component.html

```diff
@@ -1,10 +1,5 @@
-<!-- TEST ELEMENT FOR STORYBOOK-->
-<div [ngClass]="resolvePositionCases">
-  <button
-    [ngClass]="resolveButtonCases"
-    [uiTooltip]="hint"
-    class="bg-gray-200 shadow-md hover:shadow-xl hover:bg-gray-300 p-2 rounded-md"
-  >
-    Random content
-  </button>
-</div>
+<span
+  #tooltip
+  class="opacity-80 block transition-opacity delay-300 bg-gray-800 p-2 max-w-xs whitespace-pre-wrap text-xs text-gray-100 rounded-md z-[9999] break-words pointer-events-none"
```

Comment:
```suggestion
  class="block transition-opacity delay-300 bg-gray-800 p-2 max-w-xs whitespace-pre-wrap text-xs text-gray-100 rounded-md z-[9999] break-words pointer-events-none"
```
To look same as before @AntoineRelief